server/sonar-server/src/main/java/org/sonar/server/projectanalysis/ws/CreateEventAction.java

```diff
@@ -104,6 +104,8 @@ public void define(WebService.NewController context) {
 
   @Override
   public void handle(Request httpRequest, Response httpResponse) throws Exception {
+    userSession.checkLoggedIn();
```

Comment:
commit title tells of "api/projectanalysis/create" but this is "api/projectanalysis/create_event"